app/services/school_data_importer.rb

```diff
@@ -4,6 +4,10 @@
 require "csv"
 
 class SchoolDataImporter
+  def initialize(logger)
+    @logger = logger
```

Comment:
It could be just personal preference, but I find it neater and a little easier on the eyes to add attr_reader :logger and then just logger.info rather than @logger.info